src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -574,6 +574,7 @@ function _RRTMGPModel(
     use_global_means_for_well_mixed_gases::Bool = false,
     kwargs...,
 )
+    @assert pkgversion(RRTMGP) > v"0.19.2"
```

Comment:
We already have RRTMGP compat set to v0.21 in Project.toml, so maybe we can remove this line?